shared/src/business/entities/judgeActivityReport/JudgeActivityReportSearchCaseStatusSearch.ts

```diff
@@ -0,0 +1,69 @@
+import { CAV_AND_SUBMITTED_CASE_STATUS_TYPES } from '../EntityConstants';
```

Comment:
combine into JudgeActivityReport entity

Reply:
Hey @Tom Elliott, the reason the J `JudgeActivityReportSearch` and `JudgeActivityReportCaseStatusSearch` entities are separated is because combining both will require endDate and startDate filters to be optional. The Cav and Submitted cases search don't require endDate and startDate for their searches but the other reports do.

Reply:
Addressed the feedback. Thank you.